### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Hiiiij**
First milestone --> V1 to review 

Implementation of Sign Up and Sign In logic

### CODE REVIEW

File: vibezz/src/app/SignUp/page.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+'use client';
+import { useEffect, useState } from 'react';
+import { Dispatch } from 'redux';
+import { addUser } from '../store/actionCreator';
+import { useDispatch, useSelector } from 'react-redux';
+import SignIn from '../components/SignIn';
+import { redirect } from 'next/navigation'
+
+type Props = {
+  saveUser: (user: IUser | any) => void;
```
Comment (ahzia) [inline vibezz/src/app/SignUp/page.tsx:10]: try to avoid using any types.

File: vibezz/src/app/SignUp/page.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    // Will add pop up messages for wrong input when(if) we have a design
+
+    if (validation) {
+      const newUser = { userName, email };
+      dispatch(addUser(newUser));
+      dispatch({ type: 'LOGIN', user: { userName } });
+      setUserName('');
+      setEmail('');
+      console.log(`New User:`, newUser);
```
Comment (ahzia) [inline vibezz/src/app/SignUp/page.tsx:37]: Always remove the testing codes (console.log) from the code you want to merge to main

File: vibezz/src/app/components/Navbar.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    return (
+        <div>here comes the nav</div>
+    )
+} 
+
+export default Navbar;
+
+
+
+
```
Comment (ahzia) [inline vibezz/src/app/components/Navbar.tsx:13]: Remove the extra line, only one extra line is enough at the end of files

**Review by ahzia — COMMENTED** [review]
Good Job so far @bojana-petroska  @Hiiiij 👏👏👏
Everything looks good, there are a few suggestion:

- Move everything from vibezz folder to the root of the repo, it will be easier to deploy it later if you have your main files in the repository root.

- Take a look at the inline comments

File: vibezz/src/app/SignUp/page.tsx (hunk 1 above)
Comment (bojana-petroska) [reply vibezz/src/app/SignUp/page.tsx:10]: that line is already gone

File: vibezz/src/app/SignUp/page.tsx (hunk 2 above)
Comment (bojana-petroska) [reply vibezz/src/app/SignUp/page.tsx:37]: ok, thanks!

File: vibezz/src/app/components/Navbar.tsx (hunk 3 above)
Comment (bojana-petroska) [reply vibezz/src/app/components/Navbar.tsx:13]: ok

**Pull request merged**